src/seedu/addressbook/ui/TextUi.java

```diff
@@ -37,12 +31,14 @@
 
     private final Scanner in;
     private final PrintStream out;
+    private static Formatter formatter;
```

Comment:
geee.. the divider is still here :-P you need to move it to the formatter, since it is a part of the formatting